packages/components/note/src/note.js

```diff
@@ -0,0 +1,99 @@
+import { TSElement, unsafeCSS, html, customElementDefineHelper } from '@tradeshift/elements';
+import '@tradeshift/elements.icon';
+
+import { classNames, types, customEventNames } from './utils';
+import css from './note.css';
+
+customElementDefineHelper(
+	'ts-note',
+	class extends TSElement {
+		static get styles() {
+			return [TSElement.styles, unsafeCSS(css)];
+		}
+
+		static get properties() {
+			return {
+				icon: { type: String },
+				type: { type: String },
+				rtl: { type: Boolean },
```

Comment:
May be we need to put `rtl` to core, that all components can reuse it.

Reply:
Not sure about this yet! Let's decide later! Currently we have some components which not using this! and another thing is the documentation and be easy to understand! I think it's better to don't make things complex by overengineering!

Reply:
👍